channels/email/email-headers.rst

```diff
@@ -14,35 +15,25 @@ Header checks are case insensitive.
 
 .. tip::
 
-   Below headers are examples and –in our opinion– the most relevant ones.
-   However: You can adjust mostly any article or ticket attribute (yes, custom
-   ones as well) if you know the attribute's exact name.
-
-   The name column within :doc:`object's management </system/objects>` provides
-   easy access to objects attribute names. 🤓
-
-Trigger auto responses
-----------------------
+   - Below headers are examples and in our opinion the most relevant ones.
+     However, you can adjust mostly all article or ticket attributes including
+     custom ones if you know the attribute's exact name. Have a look
+     :doc:`here </system/objects>` to find the attribute names.
+   - Header checks are case insensitive. However, values are case sensitive!
```

Comment:
What do you mean that values are case sensitive?

Reply:
This comes from an old warning:
`"While headers are not case sensitive, values like e.g. priority names are case censitive: 1 low will work, but 1 lOw will not!"`

I understood it that way that header names are not case sensitive, but the values/attributes you send with them.

Reply:
Just a reword from my side: `Please note that while header names are case insensitive, header values are not. Make sure to specify values in expected case, otherwise they will not match.`

Reply:
Thank you guys!